config/environments/development.rb

```diff
@@ -67,4 +67,15 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => 'a5c8d8c55c885a',
+    :password => '975f061dea3481',
+    :address => 'smtp.mailtrap.io',
+    :domain => 'smtp.mailtrap.io',
+    :port => '2525',
+    :authentication => :cram_md5
+  }
```

Comment:
- [ ] When I try to sign up a new user it says the confirmation message has been sent to the email but I didn't get any confirmation emails. So please fix that.

![image](https://user-images.githubusercontent.com/68796232/195873272-7d69e4dd-9299-4a24-8675-28e6044e34db.png)